src/core/reference.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * @module Foundation
+ * @submodule Declaration
+ * @for p5
+ */
+
+
+/**
+
+ * Declares a block scope local variable, optionally initializing it to a value.
+ *
+ * @property let
+ * 
+ * Syntax:
+ * let var1 = value1, var2 = value2
+ * @params var1, var2, …, varN
+ * value1, value2 // Optional
+ *
+ * @example
+ * <div>
+ * <code>
+ * function letTest(){
+ * let x = 1;
+ * if(true){
+ * let x = 2; //different variable
+ * console.log(x); // 2
+ * }
+ * console.log(x); //1
+ * }
```

Comment:
So this is a bit tricky but all of the code examples have to be linted properly and they can't be autofixed. But they should all follow the formatting rules. So the code example above would become:
```
 * function letTest() {
 *   let x = 1;
 *   if (true) {
 *     let x = 2; //different variable
 *     console.log(x); // 2
 *   }
 *   console.log(x); //1
 * }
```
This can be tricky but if you run `npm run grunt` in your terminal after running `npm run lint:fix` (see final review comment) you will see a line by line breakdown of all expected formatting changes. Note that in this breakdown, a period '.' represents an empty space.

Reply:
@stalgiag 
Hi,It is very difficult to track all the changes after running the above command.